packages/testcontainers/src/containers/RegTestContainer/Masternode.ts

```diff
@@ -210,6 +224,48 @@ export class MasterNodeRegTestContainer extends RegTestContainer {
     }, timeout, 100, 'waitForPriceInvalid')
   }
 
+  /**
+   * Wait for active price
+   *
+   * @param {string} fixedIntervalPriceId
+   * @param {number} activePrice
+   * @param {number} [timeout=30000] in ms
+   * @return {Promise<void>}
+   */
+  async waitForActivePrice (fixedIntervalPriceId: string, activePrice: number, timeout = 30000): Promise<void> {
```

Comment:
duplicate of `waitForNextPrice` ?🤔

Reply:
Next price and active price are different